package.json

```
@@ -7,7 +7,9 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
```

To fix trivial errors if found:

```suggestion
    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
```